guides/events.html

```diff
@@ -62,45 +62,45 @@ <h2 id="events-and-sockets">Events and Sockets</h2>
 <p>To listen for and react to events in hsd, a listener must be added in the runtime script. If you are running a full node (for example) you might already be familiar with <a href="https://github.com/handshake-org/hsd/blob/master/bin/node">the hsd full node launch script</a>, which instantiates a <code>FullNode</code> object, adds a wallet, and begins the connection and synchronization process. The script ends with an <code>async</code> function that actually starts these processes, and this is a good place to add event listeners. <a href="#events-directory">Using the table below</a> you can discover which object needs to be listened <code>on</code> for each event type. Notice that because the wallet is added as a plugin, its object hierarchy path is a little… awkward :-)</p>
 <div class="sourceCode" id="cb1"><pre class="sourceCode javascript"><code class="sourceCode javascript"><span id="cb1-1"><a href="#cb1-1"></a><span class="co">// Based on https://github.com/handshake-org/hsd/blob/master/bin/node</span></span>
 <span id="cb1-2"><a href="#cb1-2"></a></span>
-<span id="cb1-3"><a href="#cb1-3"></a>(<span class="kw">async</span> () <span class="kw">=&gt;</span> <span class="op">{</span></span>
-<span id="cb1-4"><a href="#cb1-4"></a>  <span class="cf">await</span> <span class="va">node</span>.<span class="at">ensure</span>()<span class="op">;</span></span>
-<span id="cb1-5"><a href="#cb1-5"></a>  <span class="cf">await</span> <span class="va">node</span>.<span class="at">open</span>()<span class="op">;</span></span>
-<span id="cb1-6"><a href="#cb1-6"></a>  <span class="cf">await</span> <span class="va">node</span>.<span class="at">connect</span>()<span class="op">;</span></span>
-<span id="cb1-7"><a href="#cb1-7"></a>  <span class="va">node</span>.<span class="at">startSync</span>()<span class="op">;</span></span>
+<span id="cb1-3"><a href="#cb1-3"></a>(<span class="kw">async</span> () <span class="kw">=&gt;</span> {</span>
+<span id="cb1-4"><a href="#cb1-4"></a>  <span class="cf">await</span> node<span class="op">.</span><span class="fu">ensure</span>()<span class="op">;</span></span>
+<span id="cb1-5"><a href="#cb1-5"></a>  <span class="cf">await</span> node<span class="op">.</span><span class="fu">open</span>()<span class="op">;</span></span>
+<span id="cb1-6"><a href="#cb1-6"></a>  <span class="cf">await</span> node<span class="op">.</span><span class="fu">connect</span>()<span class="op">;</span></span>
+<span id="cb1-7"><a href="#cb1-7"></a>  node<span class="op">.</span><span class="fu">startSync</span>()<span class="op">;</span></span>
```

Comment:
hmm... why did `events.html` get re-rendered?

Reply:
I didn't change anything in the events file so I assumed that it was due to an updated dependency. It didn't look like anything broke because of it but I can remove these changes from the commits.

Reply:
I think its fine, the render looked good to me locally.